<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_change</class>
 <widget class="QMainWindow" name="page_change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>page_change</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>生成的随机访问序列：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Console:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>执行速度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="FAQ">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>FAQ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>物理块状态：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page3:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Missing Rate_FIFO:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="missingrate_FIFO">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行模式：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Model">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>660</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>作者：Warden John（张永德）</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>660</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>邮箱：309766930@qq.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIFO_show_page_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIFO_show_page_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIFO_show_page_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIFO_show_page_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page3:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LRU_page_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LRU_page_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LRU_page_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LRU_page_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LRU_show_page_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LRU_show_page_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LRU_show_page_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LRU_show_page_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行模式：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Model_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page3:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="OPT_page_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="OPT_page_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="OPT_page_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="OPT_page_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OPT_show_page_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OPT_show_page_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OPT_show_page_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OPT_show_page_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>580</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Missing Rate_LRU:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="missingrate_LRU">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行模式：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Model_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>物理块状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Console:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page3:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LFU_page0">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LFU_page0_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LFU_page0_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LFU_page0_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LFU_show_page_0">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>170</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行模式：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Model_4">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LFU_show_page_1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LFU_show_page_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LFU_show_page_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行模式：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Model_5">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>290</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Missing Rate_LFU:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="missingrate_LFU">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>620</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page3:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NRU_page0_0">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NRU_page0_1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NRU_page0_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NRU_page0_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NRU_show_page_0">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NRU_show_page_1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>340</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NRU_show_page_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NRU_show_page_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Missing Rate_NRU:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="missingrate_NRU">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>620</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>660</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>单位：北京化工大学</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>660</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyright © Warden John 2017</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>580</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Missing Rate_OPT:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="missingrate_OPT">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>580</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>****点击“start”前先输入速度！</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIFO_NEW_page">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LRU_NEW_Page">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OPT_NEW_Page">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LFU_NEW_page">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NRU_NEW_page">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>新加入的页面：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>新加入的页面：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>新加入的页面：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>新加入的页面：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>新加入的页面：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>学渣分析（想错勿喷）</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
